client/src/components/Profile/Profile.js

```diff
@@ -18,23 +20,35 @@ export default function Profile(props) {
   const [mission, setMission] = React.useState('');
   const [description, setDescription] = React.useState('');
   const [instaLink, setInstaLink] = React.useState('');
-  const [fbLink, setFbLink] = React.useState('');
+  const [fb_link, setFbLink] = React.useState('');
   const [website, setWebsite] = React.useState('');
   const [email, setEmail] = React.useState('');
   const [profilePicture, setProfilePicture] = React.useState('');
+  //add in the starred attribute
+  const [starred, setStarred] = React.useState(false);
+  const [starrerId, setStarrerId] = React.useState(null);
+  //add in the passed-in starring functionality
+  const [handleStar, setHandleStar] = React.useState(null);
 
   React.useEffect(() => {
-    setName(props.coop.name);
-    setLocation(props.coop.location.address);
+    setName(props.coop.coop_name);
+    setLocation(props.coop.addr);
+    setCoopId(props.coop.id);
+    setStarrerId(props.starrer_id);
     setPhone(props.coop.phone);
     setTags(props.coop.tags);
     setMission(props.coop.mission);
     setDescription(props.coop.description);
-    setFbLink(props.coop.fbLink);
-    setInstaLink(props.coop.instaLink);
+    setFbLink(props.coop.fb_link);
+    setInstaLink(props.coop.insta_link);
     setEmail(props.coop.email);
     setWebsite(props.coop.website);
-    setProfilePicture(props.coop.profilePicture);
+    setProfilePicture(props.coop.profile_pic);
+    //set the starred to the coop
+    setStarred(props.starred);
+    //set the handle star method
+    setHandleStar(props.handleStar);
```

Comment:
if these are never changed within the Profile component, they should just remain props. don't create a state for them